.rspec

```diff
@@ -1,2 +1,3 @@
 --color
 --require spec_helper
+-fd
```

Comment:
I'd have left this as it is — you can configure your computer to use certain rspec options globally. [See here](https://stackoverflow.com/questions/1819614/how-do-i-globally-configure-rspec-to-keep-the-color-and-format-specdoc-o).

Also it's not related to this task, so should have been a separate pull request. However, since there seems to be agreement that this is a good change then we can keep it in.